tests/metrics/machine_learning/tensorflow_mobilenet_benchmark.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o pipefial
+
+# General env
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../lib/common.bash"
+
+IMAGE="docker.io/library/tensorflowmobilenet:latest"
+DOCKERFILE="${SCRIPT_PATH}/tensorflow_mobilenet_dockerfile/Dockerfile"
+tensorflow_file=$(mktemp tensorflowresults.XXXXXXXXXX)
+NUM_CONTAINERS="$1"
+TIMEOUT="$2"
+TEST_NAME="tensorflow-intelai"
+PAYLOAD_ARGS="tail -f /dev/null"
+TESTDIR="${TESTDIR:-/testdir}"
+# Options to control the start of the workload using a trigger-file
+dst_dir="/host"
+src_dir=$(mktemp --tmpdir -d tensorflowai.XXXXXXXXXX)
+MOUNT_OPTIONS="type=bind,src=$src_dir,dst=$dst_dir,options=rbind:ro"
+start_script="mobilnet_start.sh"
```

Comment:
It doesn't truly matter since the variable and script names are consistent when they need to be, but will point out that sometimes it is spelled `mobilnet` and other times `mobilenet`.

Reply:
@cmaf thanks for the feedback, changes applied